workspaces/variables.md

```diff
@@ -57,4 +57,8 @@ env | grep CODER_
         <td><code>CODER_WP_NAME</code></td>
         <td>The name of the workspace provider hosting the environment</td>
     </tr>
+    <tr>
+        <td><code>CODER_RUNTIME</code></td>
+        <td>Whether the workspace is a CVM or not</td>
```

Comment:
Maybe

> Whether the workspace is CVM-enabled or not

WDYT?

Reply:
I think `is a` makes sense, since "containerized virtual machine" is a category, rather than something that can be turned on or off (this setting can't be changed after creation).

Reply:
Sorry that I'm late to this, CODER_RUNTIME is set to the runtime name, currently either `kubernetes/default` or `kubernetes/sysbox` (for CVMs). The goal is that we would later support other runtimes, so this isn't a boolean flag: it is the name of the container runtime that we're using to start the workspace.

Code reference: https://github.com/cdr/m/blob/5cf9288a2b3e506dfa2b31098eaf730ce99a1a4c/product/coder/pkg/model/environments_stages.go#L283-L286

Reply:
@jawnsy Thanks, I'll update this in the release branch!